### DISCUSSION TITLE
Open in new session?

### DISCUSSION THREAD

**Discussion by tamis-laan** [qutebrowser/qutebrowser#5753] [Q&A] [answered]
I would like to open a separate qutebrowser session from the command line:

`qutebrowser duckduckgo.com`

This should open in a new window and not attach to an existing qutebrowser session already running. However I can't find any option flag to do this??

**Comment by emanuele6** [comment, marked as answer]
If, with *separate session*, you mean "new window", this is how you open a URL in a new window from the command line:
```shell
$ qutebrowser --target window duckduckgo.com
```

**Reply by tamis-laan** [reply]
No, because it still attaches to an existing session... 

I need to open a completely separate session that does not closes together with the existing session.

**Reply by emanuele6** [reply]
Like a `firefox` profile?

`qutebrowser` doesn't have anything like that currently afaik.

**Reply by tamis-laan** [reply]
The problem is I'm trying to open a new separate qutebrowser session for video conferencing as followed:

`qutebrowser --target window -s statusbar.show never -s tabs.show never meet.jit.si`

Should be an entirely separate session that doesn't close with the main window and has separate options enabled.

**Reply by toofar** [reply]
You can use a separate basedir (via --basedir or -B and then a path) which will start a new instance with all the config and cache and such under the path you pass. If you don't want one that you can reuse you can pass --temp-basedir or -T and it'll use a throwaway directory under /tmp.

**Reply by tamis-laan** [reply]
Ah thnx, that works!
